framework-docs/modules/ROOT/pages/testing/testcontext-framework/tel-config.adoc

```diff
@@ -82,10 +82,10 @@ issue is addressed through support for automatic discovery of default
 
 Specifically, the `spring-test` module declares all core default `TestExecutionListener`
 implementations under the `org.springframework.test.context.TestExecutionListener` key in
-its `META-INF/spring.factories` properties file. Third-party frameworks and developers
-can contribute their own `TestExecutionListener` implementations to the list of default
-listeners in the same manner through their own `META-INF/spring.factories` properties
-file.
+[its `META-INF/spring.factories` properties file](https://github.com/spring-projects/spring-framework/blob/main/spring-test/src/main/resources/META-INF/spring.factories). 
```

Comment:
This has a `spring-framework-code` attribute that should be used rather than the hardcoded link. See `antora.yml`.

Reply:
Thanks for your feedback @snicoll, makes sense. :+1: Nice to learn about some of the Asciidoc features, seems like a nice usage of it in this case.

I adjusted both this and incorporated the suggestion from @sbrannen, please re-check. (Sam - it did perhaps not make as much sense in the https://docs.spring.io/spring-framework/reference/testing/testcontext-framework/ctx-management/context-customizers.html#testcontext-context-customizers-automatic-discovery case, since it referred to both "Spring Framework and Spring Boot", but the link only refers to the Spring Framework one.. but, possibly still a step in the right direction. :thinking:)

Reply:
Yes, indeed. That's a step in the right direction. I'll rework the wording locally after merging the PR.